pkg/custompluginmonitor/plugin/plugin_windows.go

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright 2017 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package plugin
+
+import (
+	"os/exec"
+)
+
+func execCommandInGroup(name string, arg ...string) *exec.Cmd {
```

Comment:
Does windows have the child process issue?

Reply:
See https://stackoverflow.com/questions/22470193/why-wont-go-kill-a-child-process-correctly.

People mentioned a windows solution.